lambdas/postcode-lookup/src/test/java/uk/gov/di/ipv/cri/address/api/handler/PostcodeLookupHanderTest.java

```diff
@@ -155,5 +157,6 @@ void ValidLookupReturns200() throws JsonProcessingException, SqsException {
     private void setupEventProbeErrorBehaviour() {
         when(eventProbe.counterMetric(anyString(), anyDouble())).thenReturn(eventProbe);
         when(eventProbe.log(any(Level.class), any(Exception.class))).thenReturn(eventProbe);
+        when(eventProbe.log(any(Level.class), any(String.class))).thenReturn(eventProbe);
```

Comment:
This line should be moved into a separate method (e.g. setupEventProbeDefaultBehaviour) as it is not specific to error behaviour.
The new method can then be invoked in which ever unit test requires it